rendermodules/dataimport/make_montage_scapes_stack.py

```diff
@@ -214,6 +214,32 @@ def run(self):
 
         Z = [[int(oldz),int(newz)] for oldz, newz in zip(zvalues, newzvalues)]
 
+        out_stack_exists = self.args['output_stack'] in self.render.run(renderapi.render.get_stacks_by_owner_project)
+
+        if out_stack_exists:
+            # check whether overwrite z is set to false. If so, then remove those z that is already in output stack
+            outzvalues = renderapi.stack.get_z_values_for_stack(
+                            self.args['output_stack'],
+                            render=self.render)
+            
+            if self.args['overwrite_z'] is False:
+                # do not include those sections that are already in the output stack
+                Z = [[int(oldz), int(newz)] for oldz, newz in zip(zvalues, newzvalues) if not(float(newz) in outzvalues)]
```

Comment:
Is this the behavior we want?  The other modules treat overwrite_zlayer as optionally deleting what is there vs keeping it and overlaying (I think the idea was for multiple montage sets mapping to a single section).  There  are situations where this way would probably help more, but it is a departure.

Reply:
I agree... this comes useful when handling torn sections.. Will make the changes and commit.